runtime/frequency/src/lib.rs

```diff
@@ -507,10 +506,10 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = weights::pallet_balances::SubstrateWeight<Runtime>;
 	type MaxReserves = BalancesMaxReserves;
 	type ReserveIdentifier = [u8; 8];
-	type MaxHolds = ConstU32<0>;
-	type MaxFreezes = ConstU32<0>;
+	type MaxHolds = BalancesMaxHolds;
+	type MaxFreezes = BalancesMaxFreezes;
```

Comment:
Need more research on Locks/Freezes and how they affect time-release pallet, etc.